broccoli/packages.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+/* eslint-env node */
+const { readFileSync } = require('fs');
+const path = require('path');
+const Rollup = require('broccoli-rollup');
+const Funnel = require('broccoli-funnel');
+const findLib = require('./find-lib');
+const funnelLib = require('./funnel-lib');
+const { VERSION } = require('./version');
+const WriteFile = require('broccoli-file-creator');
+const StringReplace = require('broccoli-string-replace');
+const { RELEASE, DEBUG } = require('./features');
+const GlimmerTemplatePrecompiler = require('./glimmer-template-compiler');
+const VERSION_PLACEHOLDER = /VERSION_STRING_PLACEHOLDER/g;
+
+module.exports.routerES = function _routerES() {
+  return new Rollup(findLib('router_js', 'lib'), {
+    rollup: {
+      external: ['route-recognizer', 'rsvp'],
+      entry: 'router.js',
+      plugins: [{
+        transform(code, id) {
+          if (/[^t][^e][^r]\/router\.js$/.test(id)) {
```

Comment:
Can you make an issue for us to follow up to remove this?  It seems to me that this should be part of router_js and we shouldn't have to monkey patch like this...